java/src/main/java/org/rocksdb/TickerType.java

```diff
@@ -453,43 +454,267 @@
+     * because of expiration.
+     */
+    BLOB_DB_BLOB_INDEX_EXPIRED_SIZE((char) 0x78),
+
+    /**
+     * # of blob index evicted from base DB by BlobDB compaction filter because
+     * of corresponding file deleted.
+     */
+    BLOB_DB_BLOB_INDEX_EVICTED_COUNT((char) 0x79),
+
+    /**
+     * Size of blob index evicted from base DB by BlobDB compaction filter
+     * because of corresponding file deleted.
+     */
+    BLOB_DB_BLOB_INDEX_EVICTED_SIZE((char) 0x7A),
+
+    /**
+     * # of blob files being garbage collected.
+     */
+    BLOB_DB_GC_NUM_FILES((char) 0x7B),
+
+    /**
+     * # of blob files generated by garbage collection.
+     */
+    BLOB_DB_GC_NUM_NEW_FILES((char) 0x7C),
+
+    /**
+     * # of BlobDB garbage collection failures.
+     */
+    BLOB_DB_GC_FAILURES((char) 0x7D),
+
+    /**
+     * # of keys drop by BlobDB garbage collection because they had been
+     * overwritten.
+     */
+    BLOB_DB_GC_NUM_KEYS_OVERWRITTEN((char) 0x7E),
+
+    /**
+     * # of keys drop by BlobDB garbage collection because of expiration.
+     */
+    BLOB_DB_GC_NUM_KEYS_EXPIRED((char) 0x7F),
+
+    /**
+     * # of keys relocated to new blob file by garbage collection.
+     */
+    BLOB_DB_GC_NUM_KEYS_RELOCATED((char) 0x80),
+
+    /**
+     * # of bytes drop by BlobDB garbage collection because they had been
+     * overwritten.
+     */
+    BLOB_DB_GC_BYTES_OVERWRITTEN((char) 0x81),
+
+    /**
+     * # of bytes drop by BlobDB garbage collection because of expiration.
+     */
+    BLOB_DB_GC_BYTES_EXPIRED((char) 0x82),
+
+    /**
+     * # of bytes relocated to new blob file by garbage collection.
+     */
+    BLOB_DB_GC_BYTES_RELOCATED((char) 0x83),
+
+    /**
+     * # of blob files evicted because of BlobDB is full.
+     */
+    BLOB_DB_FIFO_NUM_FILES_EVICTED((char) 0x84),
+
+    /**
+     * # of keys in the blob files evicted because of BlobDB is full.
+     */
+    BLOB_DB_FIFO_NUM_KEYS_EVICTED((char) 0x85),
+
+    /**
+     * # of bytes in the blob files evicted because of BlobDB is full.
+     */
+    BLOB_DB_FIFO_BYTES_EVICTED((char) 0x86),
+
+    /**
+     * These counters indicate a performance issue in WritePrepared transactions.
+     * We should not seem them ticking them much.
+     * # of times prepare_mutex_ is acquired in the fast path.
+     */
+    TXN_PREPARE_MUTEX_OVERHEAD((char) 0x87),
+
+    /**
+     * # of times old_commit_map_mutex_ is acquired in the fast path.
+     */
+    TXN_OLD_COMMIT_MAP_MUTEX_OVERHEAD((char) 0x88),
+
+    /**
+     * # of times we checked a batch for duplicate keys.
+     */
+    TXN_DUPLICATE_KEY_OVERHEAD((char) 0x89),
+
+    /**
+     * # of times snapshot_mutex_ is acquired in the fast path.
+     */
+    TXN_SNAPSHOT_MUTEX_OVERHEAD((char) 0x8A),
 
+    TICKER_ENUM_MAX((char) 0x8B);
 
-    private final byte value;
+    private final char value;
 
-    TickerType(final byte value) {
+    TickerType(final char value) {
```

Comment:
Hmm this is going to be a breaking change... @sagar0 what's our policy here? For the constructor we could have a `byte` variant that coerces to a char, but the getValue() accessor won't support that.

Reply:
yeah, this is going to be a breaking change, and we don't allow breaking the API in minor versions. 
One BlobDB is not supported in RocksJava yet, we can may be get away by not adding BlobDB tickers and still keep within a byte for now?